test/cpp/qps/driver.cc

```diff
@@ -623,6 +632,24 @@ std::unique_ptr<ScenarioResult> RunScenario(
   // the result.
   bool client_finish_first =
       (client_config.rpc_type() != STREAMING_FROM_SERVER);
+  
+  if (!prometheus_query_time_file.empty()) {
+    std::ofstream timestamps_outfile;
+    timestamps_outfile.open(prometheus_query_time_file);
+    
+    std::stringstream start_epoch_string;
+    start_epoch_string << start_epoch_time.count();
+    timestamps_outfile << "{\"start\": " + start_epoch_string.str() << ", ";
+
+    std::stringstream end_epoch_string;
+    std::chrono::milliseconds end_epoch_time =
+        std::chrono::duration_cast<std::chrono::milliseconds>(
+            std::chrono::system_clock::now().time_since_epoch());
+    end_epoch_string << end_epoch_time.count();
+    timestamps_outfile << "\"end\": " + end_epoch_string.str() << "}\n";
+    
+    timestamps_outfile.close();
```

Comment:
Nit: No need to manually close. [The destructor will take care of that for you](https://www.cplusplus.com/reference/fstream/ofstream/close/).